src/web/templates/components/parameters.html

```diff
@@ -0,0 +1,250 @@
+<script>
+  function allowViewScenarioParameters() {
+    document
+      .getElementById("parameters-accordion")
+      .querySelectorAll(".accordion-button[disabled]")
+      .forEach((button) => {
+        button.removeAttribute("disabled")
+        button.dataset.toDisable = true
+      })
+  }
+
+  function disallowRenderLayout() {
+    document
+      .getElementById("parameters-accordion")
+      .querySelectorAll(".accordion-button[data-to-disable=true]")
+      .forEach((button) => {
+        button.setAttribute("disabled", true)
+      })
+  }
+
+  function renderLayoutPressed(button) {
+    BtnLoading(button)
+    renderLayout()
+    BtnReset(button)
+  }
+
+  function scenarioIs2d(){
+    const is2d = document.getElementById("is2d").checked
+    return is2d
+  }
+
+  function renderLayout() {
+    showLoading("Rendering layout...")
+    disallowRunSimulation()
+    clearPlotArea()
+
+    const scenarioParams = getScenarioParams()
+    const transducersParams = getTransducersParams()
+    const targetParams = getTargetParams()
+    const simulationSettingsParams = getSimulationSettingsParams()
+
+    const data = {
+      is2d: scenarioIs2d(),
+      scenarioSettings: scenarioParams,
+      transducers: transducersParams,
+      target: targetParams,
+      simulationSettings: simulationSettingsParams,
+    }
+    const url = "{{ url_for('main.render_layout') }}"
+    fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(data),
+    })
+      .then((response) => response.text())
+      .then((text) => {
+        document.getElementById("plotArea").innerHTML = text
+        allowRunSimulation()
+        hideLoading()
+      })
+  }
```

Comment:
If there is a way to surface any data-validation errors (e.g., aperture is too large) to the user, that could be helpful:

https://www.loom.com/share/27a5b9a2b1f647d28feb1e1ea254bcb1

Also, if there is a way to change the step size for some of the fields (e.g. aperture -> `1cm` increments), that could be helpful

Reply:
We will add proper validation to the fields in a follow-up story.